tools/rna_tools/targetfinder/TargetFinder.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="targetfinder" name="targetfinder" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="1.07">targetfinder</requirement>
+        <requirement type="package" version="35.4.12">fasta</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+
+    <command><![CDATA[
+        targetfinder.pl
+            -s "$s"
+            -d $d
+            -q "$q"
+            -c $c
+            -t $t
+            -p "$p"
+            $r
+            > $output1
+    ]]></command>
+    <inputs>
+        <param label="Small RNA sequence (RNA or DNA, 5'->3')" help="(-s)" name="s" type="text"/>
+        <param label="Target sequence database file (FASTA-format)" help="(-d)" name="d" type="data" format="fasta" />
+        <param label="Query sequence name (DEFAULT = 'query')" help="(-q)" name="q" type="text" value="query" />
```

Comment:
(DEFAULT = 'query') should go into the help="" text. Help can be really long and educative. label should be short and informative.